hare/eligibility/oracle.go

```diff
@@ -356,9 +351,12 @@ func (o *Oracle) CalcEligibility(
 
 	for x := 0; x < n; x++ {
 		if fixed.BinCDF(n, p, x).GreaterThan(vrfFrac) {
+			// even with large N and large P x will be << 2^16, so this cast is safe
 			return uint16(x), nil
 		}
 	}
+
+	// since BinCDF(n, p, n) is 1 for any p this code is only reached if n << 2^16
```

Comment:
i don't understand this comment. what does "if n << 2^16" mean?

Reply:
I don't know how I came to this but I meant if n is significantly smaller than 2^16. I'll update the comment.